tools/ingest2parquet/Makefile

```diff
@@ -3,22 +3,40 @@ REPOROOT=../../
 
 include $(REPOROOT)/.make.defaults
 
-DOCKER_IMAGE_NAME=ingest2parquet
-DOCKER_IMAGE_VERSION=0.1
+include ../../.make.versions
```

Comment:
this is already included by .make.defaults.

Reply:
Done